services/identity/src/main/java/com/crapi/service/Impl/UserServiceImpl.java

```diff
@@ -124,6 +114,21 @@ public JwtResponse authenticateUserLogin(LoginForm loginForm)
     return jwtResponse;
   }
 
+  /**
+   * @param verifyTokenRequest contains JWT token to be verified
+   * @return boolean with token valid or not
+   */
+  @Transactional
+  @Override
+  public CRAPIResponse verifyJwtToken(String token) {
+    boolean validToken = jwtProvider.validateJwtToken(token);
+    if (validToken) {
+      return new CRAPIResponse(UserMessage.VALID_JWT_TOKEN, 200);
+    }
+
+    return new CRAPIResponse(UserMessage.INVALID_JWT_TOKEN, 400);
```

Comment:
If token exists, it should be 401 Unauthorized

Reply:
Updated.